platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -22,12 +22,11 @@ NS_ASSUME_NONNULL_BEGIN
 #pragma mark Responding to Map Position Changes
 
 /**
- Tells the delegate that the viewpoint depicted by the map view is about to
- change.
-
- This method is called whenever the currently displayed map camera will start
- changing for any reason.
-
+ Tells the delegate that the viewpoint depicted by the map view is changing.
```

Comment:
Saying "is changing" here blurs the distinction between this method and `-mapViewRegionIsChanging:`.